Packs/USTA/Integrations/USTA/USTA.yml

```diff
@@ -118,47 +170,99 @@ script:
       - phishing_site
       - data_leak
       - clients
-    - description: Starting Date
+    - description: 'Starting Date(Example: 2021-03-18-13-59 / yy-mm-dd-hh-mm)'
       name: start
-    - description: End Date
+    - description: 'End Date(Example: 2021-03-18-13-59 / yy-mm-dd-hh-mm)'
       name: end
     description: You can access stolen customer accounts via Stolen-Client-accounts
       API.
+    outputs:
+    - contextPath: Usta.StolenClientAccounts.created
+      description: Created date
+    - contextPath: Usta.StolenClientAccounts.password
+      description: Password
+    - contextPath: Usta.StolenClientAccounts.source
+      description: Source
+    - contextPath: Usta.StolenClientAccounts.url
+      description: URL
+    - contextPath: Usta.StolenClientAccounts.username
+      description: Username  
     name: usta-get-stolen-client-accounts
   - arguments:
     - description: Search with domain name
       name: domain
       required: true
     description: If you want to get more detailed information about malicious domain
       names, you can use this command.
+    outputs:
+    - contextPath: Usta.Domain.asn_records
+      description: ASN records
+    - contextPath: Usta.Domain.country
+      description: Country
+    - contextPath: Usta.Domain.dns_records
+      description: DNS records
+    - contextPath: Usta.Domain.domain
+      description: Domain
+    - contextPath: Usta.Domain.ip_addresses
+      description: IP addresses
+    - contextPath: Usta.Domain.ssl_records
+      description: SSL records
+    - contextPath: Usta.Domain.whois_records
+      description: Whois records  
     name: usta-get-domain
   - arguments:
     - description: Search with IP Address
       name: ip_address
     description: If you want to get more detailed information about specific IP Address,
       you can use this command.
+    outputs:
+    - contextPath: Usta.IPAddress.asn_records
+      description: ASN records
+    - contextPath: Usta.IPAddress.country
+      description: Country
+    - contextPath: Usta.IPAddress.ip_address
+      description: IP address
+    - contextPath: Usta.IPAddress.ssl_records
+      description: SSL records
+    - contextPath: Usta.IPAddress.whois_records
+      description: Whois records  
     name: usta-get-ip-address
   - arguments:
     - description: 'URL Value. Example: http://www.google3.com'
       name: address
       required: true
     description: You can search about the accuracy of the urls referring to your company's
       websites.
+    outputs:
+    - contextPath: Usta.Referrer.error
+      description: If any errors are received, it gives the details of the error  
     name: usta-send-referrer-url
   - arguments:
     - description: 'Search with this identity number. You can search all identity
         number with "," '
       name: identity_number
       required: true
+      auto: PREDEFINED
+      predefined:
+      - ""
+      - ""
```

Comment:
Why are the values here empty? predefined values are for single select arguments.
No need for this, you can just add the isArray: true field

Reply:
I changed it, Actually, first I changed this parameter on XSOAR(my environment) and I downloaded yaml and this yaml file include this predifined value, and I added github yaml file. XSOAR yaml different from yaml file on Github.